test/counter.go

```diff
@@ -27,23 +27,23 @@ func UserAccountsForCluster(clusterName string, number int) ExpectedNumberOfUser
 func AssertThatUninitializedCounterHas(t *testing.T, numberOfMurs int, numberOfUasPerCluster ...ExpectedNumberOfUserAccounts) {
 	counts, err := counter.GetCounts()
 	assert.EqualErrorf(t, err, "counter is not initialized", "should be error because counter hasn't been initialized yet")
-
 	verifyCounts(t, counts, numberOfMurs, numberOfUasPerCluster...)
 }
 
 func AssertThatCounterHas(t *testing.T, numberOfMurs int, numberOfUasPerCluster ...ExpectedNumberOfUserAccounts) {
-	AssertMetricsGaugeEquals(t, numberOfMurs, metrics.MasterUserRecordGauge)
```

Comment:
May I know why did you remove it?

Reply:
I moved it in the `verifyCounts` func which is called on line 36, so the assertion on the `metrics.MasterUserRecordGauge` is done when calling both the `AssertThatCounterHas` and the `AssertThatUninitializedCounterHas` funcs